src/scenes/Modals/SettingsModal/SettingsModal.tsx

```diff
@@ -240,8 +240,6 @@ function SettingsModal({
 }
 
 const StyledManageWalletsModal = styled(VStack)`
-  width: 300px;
```

Comment:
this was making the content artificially narrow on small mobile screens. on mobile, the popover width is dictated by the parent containers so this component doesn't have to set a width